packages/documentation-v7/src/stories/components/heading/heading.docs.mdx

```diff
@@ -0,0 +1,32 @@
+import { Meta, Canvas, Controls } from '@storybook/blocks';
+import { linkTo } from '@storybook/addon-links';
+import * as HeadingStories from './heading.stories';
+
+<Meta of={HeadingStories} />
+
+# Button
+
+<div className="lead">Use headings to structure content both visually and semantically.</div>
```

Comment:
Either you have to use a `<p>` tag or you have to write with line breaks for the `<p>` to be automatically rendered.

```suggestion
<p className="lead">Use headings to structure content both visually and semantically.</p>
```